mmdet/models/dense_heads/fovea_head.py

```diff
@@ -73,6 +74,8 @@ def __init__(self,
         self.with_deform = with_deform
         self.deform_groups = deform_groups
         super().__init__(num_classes, in_channels, init_cfg=init_cfg, **kwargs)
+        # self.strides does not need to be passed in.
```

Comment:
I believe move `* stride` to `MlvlPointGenerator` instead of putting it outside like it is now can make code more clear